README.md

```diff
@@ -58,6 +67,82 @@ aws_secret_access_key=...
 aws_session_token=...
 ```
 
+### Set up using LocalStack
+
+#### Set up the LocalStack environment
+
+This setup is only recommended for development.
+
+[LocalStack](https://www.localstack.cloud/) is a set of tools that will enable you to run this example on your local machine. To install LocalStack, follow the [installation instructions](https://docs.localstack.cloud/getting-started/installation/) for your configuration.
+
+Once LocalStack is installed, open a shell window and start the LocalStack stack in detached mode:
+
+```shell
+localstack start -d
+```
+
+Make sure to keep LocalStack running while you work through this example.
+Warning: Anything configured in LocalStack by default is transient, and will be lost on restart/reboot of the LocalStack container.
+
+To facilitate interacting with LocalStack, you may want to install the CLI tool `awslocal`. `awslocal` is a LocalStack AWS CLI, that substitutes itself to the `aws` CLI while you have LocalStack running. Detailed installation instruction are found [here](https://docs.localstack.cloud/user-guide/integrations/aws-cli/).
+
+As `awslocal` is a wrapper around the `aws` cli, you need the `awscli` package installed first. Make sure your Python virtual environment is activated and run the install command:
+
+```shell
+pip install awscli
+```
+
+To install `awslocal` into your local virtual environment for this example, make sure your Python virtual environment is activated and run:
+
+```shell
+pip install awscli-local
+```
+
+#### Creating an environment file for use with LocalStack setup
+
+When LocalStack is in use, the example's code will rely on environment variables to get credentials and find the (LocalStack) endpoint to use. Therefore, we need to set up a `.env` file in the root of the repository following the format of the [example-env.env file](example-env.env). The content of your `.env` file will be automatically read by the setup script and the Flask app so they can be reused there.
```

Comment:
With LocalStack, the example uses environment variables to get credentials and the LocalStack endpoint to access.  Therefore, you need to....